src/editor/inspect.lua

```diff
@@ -37,17 +37,26 @@ if ide.config.staticanalyzer.luacheck then
   end
 
   warnings_from_string = function(src, file)
-    local data = luacheck.check_strings({src}, config.options or {
+    local api_globals = build_env()
+
+    if config.options then
+      -- add user config globals to api table
+      for k, v in pairs(config.options.globals or {}) do
```

Comment:
I think it is config.options.std.globals or {} ? (not config.options.globals or {})
https://luacheck.readthedocs.io/en/stable/config.html#custom-stds

Reply:
@paul-reilly, should I change this to `...options.std.globals`, as @jack9 suggested?

Reply:
Also, `default_options` below used to have `std.globals`, but now simply set `globals` field. Should that be updated as well?

Reply:
no, this is correct, as `std.globals` is for standard globals and `globals` is for everything else, according to my reading of the docs.